db/migrations/state/0005.sql

```diff
@@ -0,0 +1,20 @@
+-- +migrate Up
+DROP table state.event;
+DROP table state.debug;
+
+-- +migrate Down
+CREATE TABLE state.event
```

Comment:
I'd prefer to use `IF NOT EXIST` to avoid problems. I know this situation shouldn't happen in theory